### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by katklat**
API import and export are now functioning.
Some changes to the styling and addition of a filtered index view.

### CODE REVIEW

File: app/Http/Controllers/ApiController.php [hunk 1]
```diff
@@ -46,6 +48,7 @@ public function search(SpotifyWebAPI\SpotifyWebAPI $api, $type)
                 return $this->renderSongs($results, $query);
                 break;
             case 'playlist':
+                session(['basket' => request()->basket]);
```
Comment (tillschander) [inline app/Http/Controllers/ApiController.php:51]: I think you renamed your baskets so you maybe need to change this aswell?

File: resources/views/baskets/playlists.blade.php [hunk 2]
```diff
@@ -20,7 +20,12 @@
         </form>
         <hr>
         @endforeach
+
+    </div>
+    <div class="mt-6">
+        <a class="btn btn_cancel ml-2 " href="{{ route('baskets.show', session()->get('basket')) }}">Cancel</a>
```
Comment (tillschander) [inline resources/views/baskets/playlists.blade.php:26]: Again, if you renamed baskets to lists maybe this should be renamed aswell?

File: resources/views/baskets/index.blade.php [hunk 3]
```diff
@@ -4,15 +4,29 @@
...
+            <a class="py-2 d-md-inline-block text-white filter__link" href="{{ url('baskets') }}">all</a>
+            @endif
+            @foreach($filters as $filter)
+            @if($filter !== $selectedFilter)
+            <a class="py-2  d-md-inline-block text-white filter__link" href="{{ url("baskets/show/{$filter}") }}">{{ $filter }}</a>
+            @endif
+            @endforeach
+        </div>
+
         @foreach ($baskets as $basket)
```
Comment (tillschander) [inline resources/views/baskets/index.blade.php:19]: Didn't you rename your baskets

**Review by tillschander — APPROVED** [review]
Looks good. I have nothing to add other than being confused about your naming (lists, baskets, ...) :D

File: resources/views/baskets/index.blade.php (hunk 3 above)
Comment (katklat) [reply resources/views/baskets/index.blade.php:19]: I did. This commit will be included in the next pull request

**Pull request merged**